redux-devtools/redux-devtools.d.ts

```diff
@@ -14,24 +14,24 @@ declare module "redux-devtools" {
 declare module "redux-devtools/lib/react" {
```

Comment:
Did you try this definition with a version superior to 2.1.4 ?
Like 3.0.0-alpha3 ? https://github.com/gaearon/redux-devtools/releases